src/redux/actions/auth/authActions.js

```diff
@@ -20,9 +20,16 @@ const loginAction = ({ username, password }) => (dispatch) => {
     return dispatch(login(response));
   }).catch((error) => {
     if (error.response) {
-      return dispatch(loginFailed(error.response.data.detail));
+      let errorMessage = error.response.data.detail;
+      if (errorMessage === 'Not found.') {
+        errorMessage = 'No such user was found. Please signup instead.';
+      }
+      if (errorMessage === 'Passwords did not match.') {
+        errorMessage = 'Either the username or password you provided is incorrect.';
+      }
+      return dispatch(loginFailed(errorMessage));
     }
-    return dispatch(loginFailed('Request cannot be made'));
+    return dispatch(loginFailed('Looks like your internet connection is off.'));
```

Comment:
Since there is nothing to check of the internet connection is `on` or `off`. `Please check your internet connection` seems more friendly

Reply:
@Tittoh I have changed it to that.